extension/src/FileFunctions.ts

```diff
@@ -33,3 +34,81 @@ export function loadJson(filePath: string): unknown {
   const json = JSON.parse(stripJsonComments(fileContent));
   return json;
 }
+
+export async function zipFiles(
```

Comment:
Is async still needed here?
I don't see the await...

I think that some of the workspace functions was async at some point...

Reply:
I don't think so. Removing async.